.travis.yml

```diff
@@ -9,8 +9,6 @@ notifications:
 node_js:
   - '6'
```

Comment:
better add a test for node 8 too

Reply:
unfortunately, the tests don't work for Node 8 due to the way they've been set up